docs/tableOfContents.js

```diff
@@ -170,6 +171,27 @@ export default [
       }),
     ],
   }),
+  new Section({
+    title: 'Composables',
```

Comment:
I imagine that we'd ideally move already existing composables (responsiveWindow and responsiveElement) to this section, rather than having them hidden in the components docs, and also move their code files to the same folder. In that way, we would have consistent imports in products.

I hope having this section above components will help with discoverability of KDS Composition API utilities (see https://github.com/learningequality/kolibri/pull/11132#discussion_r1312008704)  cc @akolson

Can open follow-up.